cmd/daily-lifecycle-ops.go

```diff
@@ -57,12 +57,15 @@ func lifecycleRound(ctx context.Context, objAPI ObjectLayer) error {
 
 	for _, bucket := range buckets {
 		// Check if the current bucket has a configured lifecycle policy, skip otherwise
-		l, ok := globalLifecycleSys.Get(bucket.Name)
-		if !ok {
+		l, err := globalLifecycleSys.Get(bucket.Name)
+		if err != nil {
+			if _, ok := err.(BucketLifecycleNotFound); !ok {
+				logger.LogIf(ctx, err)
+			}
 			continue
 		}
 
-		_, bucketHasLockConfig := globalBucketObjectLockSys.Get(bucket.Name)
+		rcfg, _ := globalBucketObjectLockSys.Get(bucket.Name)
```

Comment:
We could/should probably get rid of all these individual managers and just use the global conf. Not now, though.

(You can Resolve this)

Reply:
Yeah didn't want too much disruption elsewhere.